tools/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -573,10 +557,27 @@ private String handleDayPeriod(XPathParts parts, String value) {
         //ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="stand-alone"]/dayPeriodWidth[@type="wide"]/dayPeriod[@type="morning1"]
         List<String> examples = new ArrayList<>();
         final String dayPeriodType = parts.getAttributeValue(5, "type");
+        if (dayPeriodType == null) {
```

Comment:
It would be better to skip all items ending with /alias.... Then these tests can be removed (or if there are remaining nulls, then analyzed to see what the underlying problem is.

Reply:
Do you mean that getExampleHtml would always return null right away for /alias paths? I've just tried that and it does prevent hitting null further down in this case. I'll make a new PR.